testsuite/generate-format/tests/staged/api.yaml

```diff
@@ -679,6 +679,10 @@ TransactionData:
       Pending:
         NEWTYPE:
           TYPENAME: SignedTransaction
+    2:
+      PendingQueryable:
```

Comment:
@gregnazario  not seeing any bcs serde used for transactiondata. So I assume we can add a field to this TransactionData struct? not sure if there is any other use of this struct